app/StateMachines/SiteStatusStateMachine.php

```diff
@@ -12,6 +12,7 @@ public function transitions(): array
 
         $parentTransitions[self::NEEDS_MORE_INFORMATION][] = self::RESTORATION_IN_PROGRESS;
         $parentTransitions[self::APPROVED][] = self::RESTORATION_IN_PROGRESS;
+        $parentTransitions[self::AWAITING_APPROVAL][] = self::RESTORATION_IN_PROGRESS;
```

Comment:
If a site skips straight from awaiting approval to restoration in progress, the hooks in the state machine for `approved` won't go off. See `afterTransitionHooks` in the `EntityStatusStateMachine`. Perhaps what would be better would be to transition it to approved and then to restoration in progress to make sure the transition hooks go off as expected.

Reply:
<img width="654" alt="image" src="https://github.com/wri/wri-terramatch-api/assets/26514574/67089d23-cc34-4466-97f7-ea88dbbac63b">
My understanding is that restoration in progress is an intermediate state between awaiting approval and approved.

Reply:
So there is no need for firing off approved events.

Reply:
Oh, in that case is there any need to keep `approved` -> `restoration-in-progress`?

Reply:
It is exactly the rule that we need to address the new polygons creations that will immediately change a approved site to a "restoration-in-progress" site.